### ISSUE TITLE
How to replicate results of the paper?

### ISSUE THREAD

**Issue by sarahmhooper** [koncle/CoraNet#1]
Thank you for your paper and code. I'm wondering which script to run to replicate the paper results with the proposed method? I ran train_ST_3D_VNet.py and achieved the baseline results (e.g., Dice of 69 for 12 labeled/50 unlabeled images on the pancreas CT dataset), but it's not clear which training script to run to get the best performance (e.g., Dice of 79 for 12 labeled/50 unlabeled images on the pancreas CT dataset).

**Comment by koncle** [comment]
In my implementaion, I used a learning rate of 1e-3 and pretraining epoch of 60 to train the model. I have just re-run the code and get 79.72% accuracy. I have fixed some error and updated  these hyper-parameters in `train_ST_3D_VNet.py`.
Hope this would help you.
